decidim-proposals/config/locales/en.yml

```diff
@@ -250,6 +253,8 @@ en:
           send: Send
           title: Edit proposal
         edit_draft:
+          discard: Discard this draft
+          discard_confirmation: Are you sure to discard this proposal draft?
```

Comment:
`Are you sure you want to discard this proposal draft?` (I added `you want `, otherwise the sentence doesn't make sense)

Reply:
ok I'll update it